packages/mask/src/plugins/Wallet/Dashboard/index.tsx

```diff
@@ -14,9 +13,7 @@ const dashboard: Plugin.Dashboard.Definition = {
             <>
                 <ConnectWalletDialog />
                 <GasSettingDialog />
-                {getSiteType() !== ExtensionSite.Popup ? (
-                    <TransactionSnackbar pluginID={NetworkPluginID.PLUGIN_EVM} />
-                ) : null}
+                {getSiteType() !== ExtensionSite.Popup ? TransactionSnackbar.open() : null}
```

Comment:
this doesn't look right. open() looks like a side-effect function you should not call it in the render function. if open() does not have a side-effect, please rename that function.